### ISSUE TITLE
Instructions for HTML template and CSS are not accessible

### ISSUE THREAD

**Issue by jhodgdon-drp** [DjangoGirls/tutorial#1338]
### Issue description
We should probably not suggest that people make HTML templates and CSS that violate the W3C accessibility guidelines.
https://www.w3.org/WAI/standards-guidelines/wcag/

There are two problems I saw in the tutorial so far:

a) In post_list.html, it uses an H1 tag for the page title (which is good), but then an H1 tag for the post title as well. It shouldn't. Accessibility guidelines state there should only be one H1 tag on a page. The other should be H2.
https://www.w3.org/WAI/tutorials/page-structure/headings/

b) In blog.css, it is suggested to make the H1 titles orange #FCA205. This is insufficient contrast for accessibility, when displayed on a white background (or when white text is displayed on this color background):
https://www.w3.org/TR/WCAG21/#contrast-minimum
https://webaim.org/resources/contrastchecker/

**Comment by jhodgdon-drp** [comment]
I looked into what it would take to fix this.
- The page "html" (Introduction to HTML) is the first place that introduces HTML, but it is fine -- it uses an H1 for the main page title, and H2 for the individual fake blog post title. So that is good -- the post_list.html file is OK at that point.
- Then on the "django_templates" page (Django Templates), when you replace this with a dynamic template, it uses H1 instead of H2 for the individual blog post titles. So, we would want to change post_list.html there (the last version of it).
- On the "css" page (CSS - make it pretty), we would need to (a) add a rule for H2 and (b) change the color in the blog.css file (shown in several places on the page). Also, post_list.html is revised in that page, the erroneous h1 section is shown several times on the page, so that would also need to change. Also there are 4 screenshots that would need to be redone with the new color: color2.png, final.png, font.png, and margin2.png.
- On the template_extending page (Template extending), you copy things from post_list.html to base.html, and those files are shown/revised in a few places, so they'll need the H1 to H2 change.
- extend_your_application also edits post_list.html, so that would need an update. And it creates a post_detal.html file, which has the same H1/H2 problem. Also screenshot images post_detail2.png and post_list2.png would need updates for the colors.
- django_forms has the same H1/H2 problem in the post_edit.html file, and screenshot images drafts.png, edit_button2.png, edit_form2.png, form_validation2.png, new_form2.png would need updates for the colors. It also shows post_detail.html, so that would need the update.

Whew. I think that's it... Any thoughts about whether this is important? I think it is... I realize it's not a tutorial about accessibility, but we should probably not start people off with suggested CSS/HTML that are not accessible.

**Comment by jhodgdon-drp** [comment]
If we did change the color... To get good contrast with white text on orange, or orange text on white, staying in the same color family, the color would need to be #6F4702 (which is brownish, not really orange any more). It won't show up as a dramatic change when you change the text to this color in the CSS topic. But any good high-contrast color will have the same problem -- if it's going on a white background, it needs to be pretty dark to have good contrast.

**Comment by jhodgdon-drp** [comment]
Just one more thought... There may be people who want to either follow the tutorial at home or attend a workshop, who have color vision or other vision problems, and who really need the color contrast to be sufficient in order to be able to see the blog. We should really try to be inclusive -- that's practically the reason for these workshops. So, I think this is important. I'll probably work up a PR soon.

**Comment by ekohl** [comment]
I'm no expert on this matter but I fully agree we should be using/recommending the best practices. It's much easier to learn it the right way than later having to relearn habits.

**Comment by ekohl** [comment]
I'm bad with colors so I'm going to defer to others.

**Comment by jhodgdon-drp** [comment]
Here's a color that still looks orange-ish to me, but passes the contrast checker for large text: #C25100

**Comment by jhodgdon-drp** [comment]
I'm concerned about making the screenshots for this issue... It looks like all of the screenshots were made on a Mac of some sort. I don't have one. So, if I made new screenshots, they would look quite different, and different from the screenshots on other pages in the tutorial. :(

So I can make a PR to change the text, but I think someone else would need to make new screenshots, on a mac, in order to keep them kind of consistent.

I'll go ahead and edit the text and make a PR for that...
